docs/guides/src/main/server/configuration.adoc

```diff
@@ -52,57 +50,53 @@ export KC_DB_URL_HOST=mykeycloakdb
-* Database specific resources are configured and prepared to run against a specific database vendor
+* Closed-world assumption about installed providers, which means that no need exists to re-create the registry at every startup
+* Configuration files are pre-parsed to reduce I/O when starting the server
+* Database specific resources are configured and prepared to run against a certain database vendor
 * By persisting build options into the server image, the server does not perform any additional step to interpret configuration options and (re)configure itself
 
-Once you run the `build` command, you won't need to set the same **build options** again when starting the server.
-
-.First stage, run the `build` command to set any build option
+.Run the `build` command to set the database to PostgreSQL before startup:
 <@kc.build parameters="--db=postgres"/>
 
-The second stage involves starting the server using any **configuration option** available from the `start` command.
+Once you run the `build` command, you do not need to set the same **build options** when you start the server.
 
-.Second stage, run the `start` command to set any configuration option when starting the server
-<@kc.start parameters="--db-url-host=keycloak-postgres --db-username=keycloak --db-password=change_me --hostname mykeycloak.acme.com"/>
+[discrete]
+=== The second stage
 
-At this stage, you are free to set any value you want to any of the configuration options.
+The second stage involves starting the server by using the `start` command and including **configuration options** with the appropriate values.
 
-Be aware that you need to escape characters when invoking commands containing special shell characters such as `;` using the CLI, so you might want to set it in the configuration file instead.
+.Run the `start` command to start the server while setting various database options
+<@kc.start parameters="--db-url-host=keycloak-postgres --db-username=keycloak --db-password=change_me --hostname mykeycloak.acme.com"/>
 
-== Configuring the server for an optimal startup time
+Note that if you invoke commands containing special shell characters such as `;` using the CLI, you need to escape those characters.  In that situation, you might choose to use the `keycloak.conf` file to set configuration options instead.
 
-In addition to the optimizations performed when you run the `build` command, you might want to avoid using CLI options when running the
-`start` command in favor of using environment variables or configuration options from the `conf/keycloak.conf` file.
+== Optimization by using a configuration file
+
+In addition to using the `build` command instead of the `start` command for optimization, a second alternative exists. You can use the `conf/keycloak.conf` file to set environment variables or configuration options. For example, you might want to avoid using CLI options when running the `start` command.
```

Comment:
```suggestion
Most optimiziations to startup and memory footprint can be achieved by using the `build` command. Additionally, you can use the `conf/keycloak.conf` file to set  configuration options. Using this file avoids some necessary parsing steps when providing configuration options using the CLI or environment variables.
```

Reply:
The edit did not make much sense because it changed the previously right contents. Tried  to come up with a better one.

Reply:
Okay, thanks, Dominik for the correction.